### PR TITLE
RXR-462: absolute/relative conversion

### PR DESCRIPTION
**Pull request by jasmineirx**
Made a function since we do this somewhat regularly and I wanted to access it within insightrxr.

The decision to return a named list was to play nicely with insightrxr:::calc_derived_covariates

### CODE REVIEW

File: R/absolute2relative.R [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#'
+#' @param quantity quantity expressed in absolute units
+#' @param bsa ideal body weight in kg
+#' @param ... arguments passed on to `calc_bsa`, if bsa is NULL
+#' @return quantity expressed relative to /1.73m2
+#' @examples
+#' absolute2relative(quantity = 60, bsa = 1.6)
+#' absolute2relative(quantity = 60, weight = 14, height = 90, method = "dubois")
+#' @export
+absolute2relative <- function(quantity, bsa = NULL, ...) {
```
Comment (dominic-irx) [inline R/absolute2relative.R:13]: I doubt this is the only thing we do an absolute to relative transformation on, so suggest adding bsa in there? ie absolute2relative_bsa or something like that.

**Review by dominic-irx — APPROVED** [review]
lgtm, one suggestion on function naming

File: R/absolute2relative.R (hunk 1 above)
Comment (jasmineirx) [reply R/absolute2relative.R:13]: what else are you thinking of? the name was already rather long.

Comment (dominic-irx) [reply R/absolute2relative.R:13]: Scaling to body weight, for example. A function named absolute2relative implies doing more than BSA abs/rel conversion.

Comment (jasmineirx) [reply R/absolute2relative.R]: i know its possible but i haven't encountered it before. i renamed it.

**Pull request merged**